data/human/jobs.txt

```diff
@@ -3781,3 +3781,1011 @@ mission "Hunted"
+	passengers 2 8 .8
+	to offer
+		random < 60
+	source
+		government "Pirate"
+		not attributes "slavery ban"
+	destination
+		government "Humanika (Port)"
+		attributes "slavery ban"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 7000 170
+		dialog phrase "generic completed emancipation mission"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+mission "Escaped Slaves [1]"
+	name "Help slaves escape to <planet>"
+	job
+	repeat
+	description "An anonymous group will pay <payment> to transport <bunks> slaves to a new life on <planet>. Expect bounty hunters."
+	passengers 4 15 .8
+	to offer
+		random < 50
+		"passenger space" > 5
+	source
+		government "Pirate"
+		not attributes "slavery ban"
+	destination
+		government "Humanika (Port)"
+		attributes "slavery ban"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 7000 170
+		dialog phrase "generic completed emancipation mission"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+mission "Escaped Slaves [2]"
+	name "Help slaves escape to <planet>"
+	job
+	repeat
+	description "An anonymous group will pay <payment> to transport <bunks> slaves to a new life on <planet>. Expect bounty hunters."
+	passengers 7 28 .8
+	to offer
+		random < 40
+		"passenger space" > 10
+	source
+		government "Pirate"
+		not attributes "slavery ban"
+	destination
+		government "Humanika (Port)"
+		attributes "slavery ban"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 7000 170
+		dialog phrase "generic completed emancipation mission"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+mission "Emancipation Raid Escort [0]"
+	name "Emancipation escort: <planet stopovers>"
+	job
+	repeat
+	description "Escort a Nighthawk to <stopovers> for an emancipation raid, then back to <planet> for <payment>. You will have a passcode for landing clearance. Beware Syndicate and bounty hunters."
+	clearance "You provide a passcode given to you on <origin> and the authorities give you landing coordinates."
+	to offer
+		"combat rating" > 50
+		random < 40
+	source
+		government "Humanika (Port)"
+		attributes "slavery ban"
+	stopover
+		government "Pirate"
+		not attributes "slavery ban"
+		attributes spaceport
+		distance 5 16
+	on visit
+		dialog "The emancipation raid isn't complete. The Nighthawk must reach <stopover planets> and then <planet>."
+	on stopover
+		dialog "The Nighthawk lands on a different vector. You can see a bunker explode from bombardment before it lands. Not long after, it meets you in the spaceport, filled with freed slaves."
```

Comment:
Landing vector. It is a piloting term.

Reply:
In a game about piloting a ship that frequently lands, I think it is reasonable to use piloting jargon. Either the player should already know it, be able to infer it, or be interested in learning the jargon. (Or simply not care, because they don't read the dialog anyway.)